src/dbnode/storage/database.go

```diff
@@ -340,6 +358,32 @@ func (d *db) addNamespacesWithLock(namespaces []namespace.Metadata) error {
 	return nil
 }
 
+func (d *db) updateNamespaceSchemasWithLock(schemaUpdates []namespace.Metadata) error {
+	for _, n := range schemaUpdates {
+		// ensure namespace exists
+		existing, ok := d.namespaces.Get(n.ID())
+		if !ok { // should never happen
+			return fmt.Errorf("non-existing namespace marked for schema update: %v", n.ID().String())
+		}
+		currentId := "none"
```

Comment:
Can we stick with either `current` or `existing` for variables in this function, and also can we call this variable `currentSchemaID` or `existingSchemaID` since ID could be misconstrued as the namespace ID

Reply:
ok